CMakeLists.txt

```diff
@@ -247,6 +249,31 @@ configure_package_config_file(
 # TODO(umar): Disable for now. Causing issues with builds on windows.
 #export(PACKAGE ArrayFire)
 
+if (USE_CPU_MKL OR USE_OPENCL_MKL)
+  set(mkl_rt_library "${CMAKE_SHARED_LIBRARY_PREFIX}mkl_rt${CMAKE_SHARED_LIBRARY_SUFFIX}")
+  find_file(MKL_RT_LIB ${mkl_rt_library}
+    HINTS
+      ENV INTEL_MKL_ROOT_DIR
+    )
+
+  get_filename_component(MKL_RT_PATH "${MKL_RT_LIB}" REALPATH)
+  get_filename_component(MKL_RT_DIR "${MKL_RT_PATH}" DIRECTORY)
+
+  file(GLOB mkl_compiler_dlls "${MKL_RT_DIR}/../compiler/*.dll")
+  file(GLOB mkl_compiler_pdbs "${MKL_RT_DIR}/../compiler/*.pdb")
```

Comment:
These should only be installed if we are shipping with debug symbols

Reply:
I found this pdb file in old installer files, so added it to be consistent.